### DISCUSSION TITLE
Este contacto del Teams, es fake ?

### DISCUSSION THREAD

**Discussion by MONZONPUNTOEXE** [Launch-X-Latam/MisionFrontEnd#489] [Team Posts]
Hola.
Me llego el mail para estar en el teams de Backend pero nose si es fake o es el oficial.
Este es el contacto:
Manager@innovaccion.mx

**Comment by AleBT75** [comment]
Si, es oficial, de esa dirección de correo me llegó a mi y ya estoy en el grupo de Teams de BackEnd
